src/components/header.js

```diff
@@ -4,6 +4,12 @@ import { StaticImage } from "gatsby-plugin-image"
 import Icon from "./Icon"
 import useLockedBody from "../hooks/useLockedBody"
 import { useHeaderData } from "../fetchHooks/useHeader"
+import { Helmet } from "react-helmet"
+
+
+<Helmet>
+       <meta name="trustpilot-one-time-domain-verification-id" content="b8bc146e-8e9b-4c23-b6c3-14020d16a77e"/> 
```

Comment:
isn't <meta> that you've added in seo enough?

Reply:
You're right. Removed from the header.js